Rules/CTF/Scripts/CTF.as

```diff
@@ -603,42 +603,56 @@ shared class CTFCore : RulesCore
 			}
 		}
 
-		if(stalemate){
-			if(!rules.exists("stalemate_breaker")){
-				rules.set_s16("stalemate_breaker",stalemateTime);
-			} else {
-				rules.sub_s16("stalemate_breaker",1);
+		if(stalemate)
+		{
+			if(!rules.exists("stalemate_breaker"))
+			{
+				rules.set_s16("stalemate_breaker", stalemateTime);
+			} 
+			else 
+			{
+				rules.sub_s16("stalemate_breaker", 1);
 			}
 			
-			if((rules.get_s16("stalemate_breaker")/30) > 0){
+			if((rules.get_s16("stalemate_breaker")/30) > 0)
+			{
 				rules.SetGlobalMessage("Stalemate: both teams have no flags.\nFlags respawning in: "+(rules.get_s16("stalemate_breaker")/30));
-			} else {
-				rules.set_s16("stalemate_breaker",-60);
+			} 
+			else 
+			{
+				rules.set_s16("stalemate_breaker", -60);
 				
-				for (uint i = 0; i < flags.length; i++)
+				if(isServer())
 				{
-					CBlob @flag = flags[i];
-					if (flag !is null)
+					for (uint i = 0; i < flags.length; i++)
 					{
-						flag.Tag("stalemate_return");
-						if(isServer())flag.Sync("stalemate_return",true);
+						CBlob @flag = flags[i];
+						if (flag !is null)
+						{
+							flag.Tag("stalemate_return");
+						}
 					}
 				}
 			}
 			
-		} else {
+		} 
+		else 
+		{
 			int stalemate_timer = rules.get_s16("stalemate_breaker");
-			if(stalemate_timer > -10 && stalemate_timer != stalemateTime){
+			if(stalemate_timer > -10 && stalemate_timer != stalemateTime)
+			{
 				rules.SetGlobalMessage("");
-				rules.set_s16("stalemate_breaker",stalemateTime);
-			} else
-			if(stalemate_timer <= -10){
+				rules.set_s16("stalemate_breaker", stalemateTime);
+			} else ///??? how the hell are elses stylised if not like this?
+			if(stalemate_timer <= -10)
```

Comment:
(for the record)
```cpp
} else if (condition) {
```

Reply:
If this is a solution to it, I'm quite sad

Reply:
It's the kind of thing [people have a lot of opinions about](https://en.wikipedia.org/wiki/Indentation_style). KAG source "canonically" (if you can really call it that) uses Allman style, likely inherited from MM's pascal roots.